.github/workflows/build-test-package.yml

```diff
@@ -8,31 +8,31 @@ jobs:
     strategy:
       max-parallel: 3
       matrix:
-        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        os: [ubuntu-18.04, windows-2022, macos-10.15]
```

Comment:
I wonder whether we still need a customized workflow file here. Maybe we can replace it by one like this: https://github.com/InsightSoftwareConsortium/ITKMontage/commit/07ff68f05392b50bccd03cfd8dec5c6a15ac2ad1

Reply:
I am trying this.  @dzenanz   I'll have limited time to keep pushing this forward, so feel free to take over any of these PR's (or components of the PR) that you find useful.  :).

Hans

Reply:
Probably not, but I don't know what to do.

Reply:
Python packaging disabled.